src/index.ts

```diff
@@ -1 +1,9 @@
+export * from '@hyperproof/hypersync-models';
+
+export {
+  CredentialFieldType,
+  CustomAuthCredentials,
+  Logger
+} from '@hyperproof/integration-sdk';
```

Comment:
Selective re-export here because there is one conflicting type in both child packages.  That will be resolved when we implement `@hyperproof/integation-models`